src/Docker.DotNet.X509/Docker.DotNet.X509.csproj

```diff
@@ -8,4 +8,10 @@
   <ItemGroup>
     <ProjectReference Include="..\Docker.DotNet\Docker.DotNet.csproj" />
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Update="Nerdbank.GitVersioning" Version="3.3.37">
```

Comment:
Please remove this. It is added by the .props file to all projects.

Reply:
After removing these, tests are failing in CI/CD pipeline and also 'dotnet build' from command line